charts/tfm-almacar/values.yaml

```diff
@@ -13,6 +13,8 @@ global:
     port: 3306
 
 kafka:
+  headless:
```

Comment:
This was not needed before ?

Reply:
No, I just put that value to let the people know. Even I am putting the default value.

It can be removed if you want.